### DISCUSSION TITLE
Welcome to ActionPlansV4 Discussions!

### DISCUSSION THREAD

**Discussion by dschach** [SalesforceLabs/ActionPlansV4#57] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by senamo** [comment]
Hi how so I assign the action plan to an existing custom object? The steps provided are for a new custom object. what is that object is a managed package? Is it unusable?
